Add link redirecting to council/motions on bounty list when voting on bounty

ekowalsk opened the pull request:
![image](https://user-images.githubusercontent.com/30404388/107024504-6c0ead80-67a8-11eb-8c3e-489ab1db9625.png)

jacogr commented on packages/page-bounties/src/Voting/VotingLink.tsx:
```diff
@@ -0,0 +1,53 @@
+}
+
+export default React.memo(styled(VotingLink)`
+  && .voting-link {
+    font-size: 0.85rem;
+    line-height: 0.5rem;
+    color: currentColor;
+    :hover {
+      color: currentColor;
+    }
+  }
```
The links should just be links (as it is everywhere atm) - the button-like appearance imho probably drives away from "open-in-new-window".

ekowalsk commented:
I'm very, very sorry for the force-push again. There's no new changes

jacogr commented:
Old habits die hard :)

jacogr reviewed (approved):
Looks good, has conflicts.

ekowalsk commented:
Hi, I've added one more commit with improvements of css, would you like to have a look at it? Or it's ok if we merge it? I've also updated the image of how it looks like now

jacogr commented on packages/page-bounties/src/Voting/VotingLink.tsx:
```diff
@@ -10,7 +10,7 @@ function VotingLink (): React.ReactElement<string> {
 
   return (
-    <div>
+    <div style={{ margin: ' 0 1rem 0 0' }}>
```
Would really prefer not to do this. This is a nightmare to maintain.

ekowalsk replied:
ok, fixed :)
